Git-cheatsheet.md

```diff
@@ -0,0 +1,71 @@
+# Git CheatSheet
+
+We use git to version control (save) our changes. This allows everyone to receive the latest changes that have been agreed by everyone. Here is a simple cheatsheet to follow to save your changes.
+
+### Pushing your changes
+
```

Comment:
I'm wondering if the git commands below could do with some more explanation? Although does depend on how much detail we want here.

One thing that might be worth indicating is this creates a new branch, so branch_name is of own choosing. This could also be pointed out in the Example further down.

Reply:
Good point 👍 Added this info to the table below where the descriptions are given and other places.